NDB.Covid19/NDB.Covid19/WebServices/ExposureNotification/ExposureNotificationWebService.cs

```diff
@@ -27,8 +34,16 @@ public async Task<bool> PostSelvExposureKeys(IEnumerable<ExposureKeyModel> tempo
 
         public async Task<bool> PostSelvExposureKeys(SelfDiagnosisSubmissionDTO selfDiagnosisSubmissionDTO, IEnumerable<ExposureKeyModel> temporaryExposureKeys, BaseWebService service)
         {
-            ApiResponse response = await service.Post(selfDiagnosisSubmissionDTO, Conf.URL_PUT_UPLOAD_DIAGNOSIS_KEYS);
-            
+            ApiResponse response;
+            if (AuthenticationState.PersonalData?.AnonymousTokensEnabled == true)
+            {
+                response = await PostSelvExposureKeysWithAnonTokens(selfDiagnosisSubmissionDTO, temporaryExposureKeys, service);
```

Comment:
mixed norwegian and english?

Reply:
Probably Danish and Norwegian, but yeah. I kept the original typo on line 43 for consistency over correctness.